Description: Set the version of the Groovy artifacts (can be removed once groovy-bom is packaged)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/pom.xml
+++ b/pom.xml
@@ -205,16 +205,19 @@
     <dependency>
       <groupId>org.apache.groovy</groupId>
       <artifactId>groovy</artifactId>
+      <version>debian</version>
       <scope>runtime</scope>
     </dependency>
     <dependency>
       <groupId>org.apache.groovy</groupId>
       <artifactId>groovy-json</artifactId>
+      <version>debian</version>
       <scope>runtime</scope>
     </dependency>
     <dependency>
       <groupId>org.apache.groovy</groupId>
       <artifactId>groovy-xml</artifactId>
+      <version>debian</version>
       <scope>runtime</scope>
     </dependency>
 
